.github/workflows/sorbet.yml

```diff
@@ -46,6 +46,7 @@ jobs:
       - name: Update RBI files
         id: update
         working-directory: ${{ steps.set-up-homebrew.outputs.repository-path }}
+        continue-on-error: true
```

Comment:
It's probably better to simply change `safe_system` to `system` for the spoom bumps otherwise we may be ignoring a broken CI here.

Reply:
```suggestion
```